apps/mymedicare_cb/views.py

```diff
@@ -24,6 +25,7 @@
 from apps.fhir.bluebutton.exceptions import UpstreamServerException
 
 logger = logging.getLogger('hhs_server.%s' % __name__)
+authenticate_logger = logging.getLogger('audit.authenticate.sls')
```

Comment:
Added `authentication_logger` so that audit info is routed to our /var/log/pyapps/perf_mon.log file on running instances.